src/test/java/fr/esgi/al5_2/Tayarim/ProprietaireControllerTest.java

```diff
@@ -36,279 +37,349 @@
+        .contentType(MediaType.APPLICATION_JSON)
+        .content(objectMapper.writeValueAsString(proprietaireCreationDTO)));
+
+    List<String> errors = List.of("error_owner_invalid_name");
+
+    response.andExpect(MockMvcResultMatchers.status().isBadRequest())
+        .andExpect(MockMvcResultMatchers.jsonPath("$.errors", CoreMatchers.is(errors)));
+  }
+
+  @Test
+  public void ProprietaireController_CreerProprietaire_ReturnErrorOwnerInvalidFirstName()
+      throws Exception {
+    proprietaireCreationDTO.setPrenom("");
+    given(proprietaireService.creerProprietaire(ArgumentMatchers.any())).willAnswer(
+        invocationOnMock -> proprietaireDTO);
+    when(authService.verifyToken(any(), anyBoolean())).thenReturn(
+        new AbstractMap.SimpleEntry<>(proprietaireDTO.getId(), true)); //skip token verif
+
+    ResultActions response = mockMvc.perform(post("/proprietaires")
+        .header("Authorization",
+            "Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJqb2huZG9lQGdtYWlsLmNvbTs2MjZjMjlhNS0zZGU0LTQ5Y2YtODI4ZS1hNDkxNWQzMzM5N2EiLCJpYXQiOjE3MTQxMTM1NzcsImV4cCI6MTcxNDExNzE3N30.kbU1pVkUHkRXktX44JBFN_xzDv-ZvSmtPjnjORO0vHPiHd3f2MGfDF15VTFO5icIrU_bV9cTqZ70RDlKdlp0-g")
+        .contentType(MediaType.APPLICATION_JSON)
+        .content(objectMapper.writeValueAsString(proprietaireCreationDTO)));
+
+    List<String> errors = List.of("error_owner_invalid_firstName");
+
+    response.andExpect(MockMvcResultMatchers.status().isBadRequest())
+        .andExpect(MockMvcResultMatchers.jsonPath("$.errors", CoreMatchers.is(errors)));
+  }
+
+  @Test
+  public void ProprietaireController_CreerProprietaire_ReturnErrorOwnerInvalidMail_WhenEmailIsEmpty()
+      throws Exception {
+    proprietaireCreationDTO.setEmail("");
+    given(proprietaireService.creerProprietaire(ArgumentMatchers.any())).willAnswer(
+        invocationOnMock -> proprietaireDTO);
+    when(authService.verifyToken(any(), anyBoolean())).thenReturn(
+        new AbstractMap.SimpleEntry<>(proprietaireDTO.getId(), true)); //skip token verif
+
+    ResultActions response = mockMvc.perform(post("/proprietaires")
+        .header("Authorization",
+            "Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJqb2huZG9lQGdtYWlsLmNvbTs2MjZjMjlhNS0zZGU0LTQ5Y2YtODI4ZS1hNDkxNWQzMzM5N2EiLCJpYXQiOjE3MTQxMTM1NzcsImV4cCI6MTcxNDExNzE3N30.kbU1pVkUHkRXktX44JBFN_xzDv-ZvSmtPjnjORO0vHPiHd3f2MGfDF15VTFO5icIrU_bV9cTqZ70RDlKdlp0-g")
+        .contentType(MediaType.APPLICATION_JSON)
+        .content(objectMapper.writeValueAsString(proprietaireCreationDTO)));
+
+    List<String> errors = List.of("error_owner_invalid_mail");
+
+    response.andExpect(MockMvcResultMatchers.status().isBadRequest())
+        .andExpect(MockMvcResultMatchers.jsonPath("$.errors", CoreMatchers.is(errors)));
+  }
+
+  @Test
+  public void ProprietaireController_CreerProprietaire_ReturnErrorOwnerInvalidMail_WhenEmailDoesntMatchFormat()
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 111).